hal/lib/athena/AnalogInput.cpp

```diff
@@ -100,8 +100,9 @@ HAL_Bool HAL_CheckAnalogInputChannel(int32_t channel) {
  * @param samplesPerSecond The number of samples per channel per second.
  */
 void HAL_SetAnalogSampleRate(double samplesPerSecond, int32_t* status) {
-  // TODO: This will change when variable size scan lists are implemented.
-  // TODO: Need float comparison with epsilon.
+  // TODO(unknown): This will change when variable size scan lists are
```

Comment:
I don't like this cpplint requirement to label TODOs.  We should disable this cpplint warning instead.